Portable.Ninject.nuspec

```diff
@@ -19,6 +19,22 @@
     <summary>IoC container for .NET</summary>
     <releaseNotes>Xamarin support for iOS and Android</releaseNotes>
     <tags>Ninject ioc di portable pcl xamarin ios android monotouch monoandroid</tags>
+    <dependencies>
+      <group targetFramework="dotnet">
```

Comment:
I actually tried. But for some reason it didn't work. `NuSpec.ReferenceGenerator` wasn't getting added to the `project.json`.  I didn't have time to look into it yet.